src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const declarations = sourceString
+    .split(/;(?![^(]*\))/)
+    .map((decl) => decl.trim())
+    .filter((decl) => decl);
+
+  declarations.forEach((declaration) => {
+    const [property, ...valueParts] = declaration.split(':');
```

Comment:
The 'split' operation assumes that there will be only one ':' character separating property and value. If the value contains additional ':' characters (e.g., in a data URI), this could lead to incorrect results.

Reply:
done